### ISSUE TITLE
diffusion connectivity - additional integration needed?

### ISSUE THREAD

**Issue by cmgtam** [uw-cmg/MAST#356]
Found these comments in Zhewen's documentation:
Currently the code is essentially standalone. The parsing of $site from the input should be integrated in MAST’s inputparsers.py and added to the <InputOption>.
In normal MAST calculations, all the defect sites that are waited to be calculated in MAST input file are given explicitly by the user. However, in connectivity, the defect sites are “overwhelming” and no neb section is given. 
So I think the connectivity part should have its own MAST recipe. create_paths.py and NEBcheck.py should merge into a run_mast_method in the inducedefect ingredient. This method first picks up all possible paths, then outputs these paths in the neb ingredient and finally MAST will generate the folders accordingly. 
After the neb and phonon calculations, there should be a checker to check if a certain path is viable and return related messages.

In a nutshell, the connectivity itself is a complete MAST calculation workflow. The major difference is that it has much more defect sites and no neb sections, which are yet to be determined from the run_mast_method.
